src/core.js

```diff
@@ -106,6 +133,8 @@ extend( QUnit, {
 
 	only: only,
 
+	on: on,
```

Comment:
Yes, but I would leave it like this, to be equal to the others props, this can be changed in another pr.